go-controller/pkg/ovn/controller/services/services_controller.go

```diff
@@ -650,21 +682,50 @@ func (c *Controller) queueServiceForEndpointSlice(endpointSlice *discovery.Endpo
 		return
 	}
 
-	c.queue.Add(key)
+	c.queue.Add(FormatKeyForService(serviceName, endpointSlice.Namespace))
```

Comment:
Maybe. But if we want to use namespaced names everywhere, also skipService will take that as input and as a consequence the controller queue should also be made of namespaced names instead of strings I'm not against it, but maybe there was a reason why we're just using strings?

```
func (c *Controller) onServiceAdd(obj interface{}) {
	key, err := cache.MetaNamespaceKeyFunc(obj)
	if err != nil {
		utilruntime.HandleError(fmt.Errorf("couldn't get key for object %+v for network=%s: %v", obj, c.netInfo.GetNetworkName(), err))
		return
	}

	service := obj.(*v1.Service)
	if c.skipService(service.Name, service.Namespace) {
		return
	}
	metrics.GetConfigDurationRecorder().Start("service", service.Namespace, service.Name)
	klog.V(5).Infof("Adding service %s for network=%s", key, c.netInfo.GetNetworkName())
	c.queue.Add(key)
}

```